### PR TITLE
start stop k3d scripts

### PR DESCRIPTION
**Pull request by vstdolittle**
## Summary

Make it easier for developers to work for short periods on studio. By having a start/stop script one can save the state in studio under development and get back to it at a later stage. reset-local-k3d would clear all volumes.

### Added

- start-local-k3d
- stop-local-k3d

### Changed

- Added documentation for the new scripts

### CODE REVIEW

File: Documentation/k3d.md [hunk 1]
```diff
@@ -18,7 +18,17 @@ Run the reset-local-k3d script, that automatically deletes the existing k3d clus
 ./Environment/k3d/reset-local-k3d
 ```
 
-For manual steps, continue following the guide.
+## Start the cluster
+> NOTE: This requires the cluster to be setup in order to work. Follow either the manual steps or run reset-local-k3d
+```sh
+./Environment/k3d/start-local-k3d
```
Comment (joelhoisko) [inline Documentation/k3d.md:24]:
```suggestion
k3d cluster start dolittle-dev
```

We actually only need one line in the script, the `k3d` command works globally so we don't have to `cd` anywhere. So I don't think that you even need the separate script files, just documenting the `k3d cluster start/stop dolittle-dev` is enough.

File: Documentation/k3d.md [hunk 2]
```diff
@@ -18,7 +18,17 @@ Run the reset-local-k3d script, that automatically deletes the existing k3d clus
...
+```
+## Stop the cluster
+```sh
+./Environment/k3d/stop-local-k3d
```
Comment (joelhoisko) [inline Documentation/k3d.md:28]:
```suggestion
k3d cluster stop dolittle-dev
```

Same thing here, it's enough to just document the command.

**Review by joelhoisko — CHANGES_REQUESTED** [review]
Thanks for the PR, docs and helper scripts are always nice. In this case I don't think that we actually need the scripts in their current form as they are just oneliners after all. If the scripts did a bit more (like an intelligent start/stop based on the state of the cluster) it would make sense to have them, but otherwise they act just as an extra layer of abstraction.

**Comment by jakhog** [comment]
As this PR has been laying dead here for a while now, I've taken the liberty of applying Joels suggestion - and I will get them in. Thanks Victor!

**Pull request merged**